playground/nextjs/src/pages/render-notification/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { novuConfig } from '@/utils/config';
+import { Inbox } from '@novu/react';
+
+export default function Home() {
+  return (
+    <div className="text-center flex flex-col gap-4">
+      <p className="text-xl">Render Notification Props</p>
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** Seems like a `h*` element would be more semantic / accurate here. Same goes for all other playground pages‏

Reply:
Agree 👍🏼